recipes/gromacs/4.6.5/meta.yaml

```diff
@@ -0,0 +1,27 @@
+about:
+  home: http://www.gromacs.org/
+  license: GNU Lesser General Public License (LGPL)
+  summary: GROMACS is a versatile package to perform molecular dynamics.
+
+package:
+  name: gromacs
+  version: 4.6.5
+
+build:
+  number: 0
+  skip: True #  [osx]
+
+source:
+  fn: gromacs-4.6.5.tar.gz
+  url: ftp://ftp.gromacs.org/pub/gromacs/gromacs-4.6.5.tar.gz
+  md5: 184a537ea52429dbc2a681f98b46ed84
+
+requirements:
+  build:
+    - gcc
+    - cmake
+  run:
```

Comment:
Hi @johanneskoester, I think so, because it's a C source code and not a pre-compiled package. In this case I need put this in runtine dependencies section? Great, I'll make this, thanks a lot!